### PR TITLE
hostnetwork and volumemount

### PR DESCRIPTION
**Pull request by Ymshero**
Signed-off-by: Yasharth <yasharthsrivastava35@gmail.com>

**What this PR does / why we need it**: 
   We have added support for hostnetwork and adding multiple volume mounts in cmdprobe.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
  #431 Multiple volumes and hostnetwork enabling was not supported earlier.
**Special notes for your reviewer**:

**Checklist:**
-   [ Y] Fixes #431
-   [ ] PR messages has document related information
-   [ ] Labelled this PR & related issue with `breaking-changes` tag
-   [ ] PR messages has breaking changes related information
-   [ ] Labelled this PR & related issue with `requires-upgrade` tag
-   [ ] PR messages has upgrade related information
-   [ ] Commit has unit tests
-   [ ] Commit has integration tests
-   [ ] E2E run Required for the changes

### CODE REVIEW

**Comment by Ymshero** [comment]
in cmdprobe we added support for hostnetwork enabling and multiple volume mounts.

File: pkg/generic/container-kill/environment/environment.go [hunk 1]
```diff
@@ -38,6 +38,20 @@ func GetENV(experimentDetails *experimentTypes.ExperimentDetails) {
 	experimentDetails.PodsAffectedPerc, _ = strconv.Atoi(common.Getenv("PODS_AFFECTED_PERC", "0"))
 	experimentDetails.Sequence = common.Getenv("SEQUENCE", "parallel")
 	experimentDetails.Signal = common.Getenv("SIGNAL", "SIGKILL")
+	experimentDetails.HostNetwork = Getenvbool("HOST_NETWORK", "false")
```
Comment (avaakash) [inline pkg/generic/container-kill/environment/environment.go:41]: We can use strconv.ParseBool(string), no need for Getenvbool function.
`experimentDetails.HostNetwork =  strconv.ParseBool(Getenv("HOST_NETWORK", "false"))`
Ref: https://pkg.go.dev/strconv#ParseBool

File: pkg/generic/container-kill/types/types.go [hunk 2]
```diff
@@ -36,4 +36,6 @@ type ExperimentDetails struct {
 	Resources           corev1.ResourceRequirements
 	Signal              string
 	ImagePullSecrets    []corev1.LocalObjectReference
+        HostNetwork         bool
```
Comment (avaakash) [inline pkg/generic/container-kill/types/types.go:39]:
```suggestion
      HostNetwork         bool
```

File: pkg/generic/disk-fill/environment/environment.go [hunk 3]
```diff
@@ -38,6 +38,20 @@ func GetENV(experimentDetails *experimentTypes.ExperimentDetails) {
 	experimentDetails.EphemeralStorageMebibytes, _ = strconv.Atoi(common.Getenv("EPHEMERAL_STORAGE_MEBIBYTES", ""))
 	experimentDetails.TerminationGracePeriodSeconds, _ = strconv.Atoi(common.Getenv("TERMINATION_GRACE_PERIOD_SECONDS", ""))
 	experimentDetails.DataBlockSize, _ = strconv.Atoi(common.Getenv("DATA_BLOCK_SIZE", "256"))
+	experimentDetails.HostNetwork = Getenvbool("HOST_NETWORK", "false")
```
Comment (avaakash) [inline pkg/generic/disk-fill/environment/environment.go:41]: Same here, we can use strconv.ParseBool(string), no need for Getenvbool function.
experimentDetails.HostNetwork = strconv.ParseBool(Getenv("HOST_NETWORK", "false"))
Ref: https://pkg.go.dev/strconv#ParseBool

File: pkg/types/types.go [hunk 4]
```diff
@@ -83,6 +83,8 @@ type ChaosDetails struct {
 	Randomness           bool
 	Targets              []v1alpha1.TargetDetails
 	ParentsResources     []string
+        VolMount             string
```
Comment (avaakash) [inline pkg/types/types.go:86]:
```suggestion
      VolMount             string
```

**Comment by Ymshero** [comment]
I have worked on the review requests , Please review and merge.

**Comment by Ymshero** [comment]
Closing the PR due to too many changes in multiple files which might create confusions.

**Pull request closed without merging**